pkg/utils/bodystring_test.go

```diff
@@ -0,0 +1,118 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package utils
+
+import (
+	"bytes"
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/heketi/tests"
+)
+
+type testGetStringFromResponseBody struct {
+	Data string
+}
+
+func (b testGetStringFromResponseBody) Close() error { return nil }
+
+func (r testGetStringFromResponseBody) Read(p []byte) (int, error) {
+	if r.Data == "" {
+		return 0, errors.New("bzzt")
+	} else {
+		return copy(p, r.Data), nil
+	}
+}
+
+func TestGetStringFromResponseOK(t *testing.T) {
+	bodytext := "Hello, Heketi!"
+	resp := &http.Response{
+		Status:        "200 OK",
+		StatusCode:    200,
+		Body:          testGetStringFromResponseBody{bodytext},
+		ContentLength: int64(len(bodytext)),
+	}
+	s, err := GetStringFromResponse(resp)
+	tests.Assert(t, err == nil, "err is nil", err)
```

Comment:
Shouldn't the message in the Assert, after the condition, naturally state what went wrong not what was expected, or that this was expected.. e.g. `tests.Assert(t, err == nil, "Expected err == nil, got : ", err)` ?